crates/sui-replay-2/src/execution.rs

```diff
@@ -101,7 +102,7 @@ pub fn execute_transaction_to_effects(
     if let Some(trace_builder) = trace_builder_opt {
         // unwrap is safe if trace_builder_opt.is_some() holds
         let output_path = get_trace_output_path(trace_execution.unwrap())?;
-        save_trace_output(&output_path, trace_builder, env)?;
+        save_trace_output(&output_path, txn.digest, trace_builder, env)?;
```

Comment:
Not necessarily needed for this PR, but I think `DEFAULT_REPLAY_HOME` should be able to be set with an env variable eventually (e.g., with `REPLAY_HOME` or something like that).

Reply:
I don't think we use this approach universally in CLI, so let's add it if/when we feel we need it